test/blast/math/simd/RegisterMatrixTest.cpp

```diff
@@ -614,25 +612,22 @@ namespace blast :: testing
         using RM = TypeParam;
         using ET = ElementType_t<RM>;
 
-        blaze::DynamicMatrix<ET, columnMajor> A(RM::columns(), RM::columns());
-        blaze::DynamicMatrix<ET, columnMajor> B(RM::rows(), RM::columns());
+        StaticMatrix<ET, RM::columns(), RM::columns(), columnMajor> A;
+        StaticMatrix<ET, RM::rows(), RM::columns(), columnMajor> B, C;
 
-        blaze::randomize(A);
-        blaze::randomize(B);
+        randomize(A);
+        randomize(B);
 
         ET alpha {};
-        blaze::randomize(alpha);
+        randomize(alpha);
 
         RM ker;
-        ker.trmmRightLower(alpha, ptr(B), ptr(A));
+        ker.trmm(alpha, ptr(B), ptr(A), UpLo::Lower, false);
 
-        // Reset upper-triangular part
-        for (size_t i = 0; i < A.rows(); ++i)
-            for (size_t j = i + 1; j < A.columns(); ++j)
-                blaze::reset(A(i, j));
+        reference::trmm(alpha, B, A, UpLo::Lower, false, C);
 
         // TODO: should be strictly equal?
```

Comment:
You can always use integer calculations, if you want strict equality (floating point arithmetic on integers is accurate)